pkg/controllers/cassandra/util/util.go

```diff
@@ -35,6 +36,14 @@ func NodePoolResourceName(c *v1alpha1.CassandraCluster, np *v1alpha1.CassandraCl
 	return fmt.Sprintf("%s-%s", ResourceBaseName(c), np.Name)
 }
 
+func SeedProviderServiceName(c *v1alpha1.CassandraCluster) string {
+	return fmt.Sprintf("%s-seedprovider", ResourceBaseName(c))
+}
+
+func CqlServiceName(c *v1alpha1.CassandraCluster) string {
+	return fmt.Sprintf("%s-cql", ResourceBaseName(c))
+}
+
```

Comment:
Note (but not required to be fixed in this PR, as I think it's a wider problem) - I've run into problems in cert-manager where resource names become too long as we just keep appending to things. We should try and work out a 'grand unified way' to generate resource names predictably.